src/renderer/components/common/ClickOutsideDetect.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * Component to detect a click outside of component bounds
+ * https://stackoverflow.com/a/42234988/852841
+ */
+export default class ClickOutsideDetect extends Component {
```

Comment:
Do we need this? I think this can be done with `focus` and `onBlur` for a better accessibility, 
anyway this component looks useful so we might just keep it for now.